editor/plugins/sprite_frames_editor_plugin.h

```diff
@@ -49,14 +49,27 @@
 class OptionButton;
 class EditorFileDialog;
 
-class EditorSpriteFramesFrame : public Resource {
-	GDCLASS(EditorSpriteFramesFrame, Resource);
+class EditorSpriteFrame : public Resource {
+	GDCLASS(EditorSpriteFrame, Resource);
 
 public:
 	Ref<Texture2D> texture;
 	float duration;
 };
 
+class EditorSpriteFrameArray : public Resource {
+	GDCLASS(EditorSpriteFrameArray, Resource);
+
+	Vector<Ref<EditorSpriteFrame>> frames;
```

Comment:
Sorry, I worded it poorly. There is no point in having both `EditorSpriteFrame` and `EditorSpriteFrameArray` since they are not exposed. I meant something like:

```c++
class EditorSpriteFrameArray : public Resource {
    GDCLASS(EditorSpriteFrameArray, Resource);

public:
    struct Frame {
        Ref<Texture2D> texture;
        float duration;
    }
    Vector<Frame> frames;
};
```

Reply:
The main class is SpriteFramesEditor, so I didn't use EditorSpriteFrames, would be confusing.
I picked ClipboardSpriteFrames instead.